bin/gaia-marionette

```diff
@@ -64,6 +64,7 @@ if [ -z "$TEST_FILES" ]; then
       if [ -z "$HARNESS_ONLY" ]; then
         TEST_FILES=$(find $DIR/.. -path "*/apps/*/test/marionette/*_test.js" \
                               -or -path "*/dev_apps/*/test/marionette/*_test.js" \
+                              -or -path "*/tv_apps/*/test/marionette/*_test.js" \
```

Comment:
Adding this here will cause all of your tv_apps tests to be run always, both for tv marionette tests and marionette tests. I think this isn't exactly what you're looking for. 

You will probably need to check for  $GAIA_DEVICE_TYPE being "tv" and then add those test files in. Or even better may be to specifically ONLY do the TV tests when device type TV is detected.

I'm not sure what type of changes and coverage you guys are expecting here so I will be able to advise better once I know a little more.